pkg/packagekit/wix/service.go

```diff
@@ -90,6 +90,12 @@ type ServiceControl struct {
 // This is used needed to set DelayedAutoStart
 type ServiceConfig struct {
```

Comment:
Just to say this out loud... I think even without the util extension, we're _still_ using wix to install the service. Which means that uninstall will _still_ remove it and we don't have to think harder?

Reply:
Yes, this is correct. The only real difference now is that the arm64 will not set recovery actions during install, they are set when windows service starts.